### ISSUE TITLE
As a Data owner/Data manager I want to a template for upload (and provide a human-readable validation output) ..so that I can submit my data more easily

### ISSUE THREAD

**Issue by LynnHeeley** [ices-tools-dev/cetaceans#12]
original id#9

regards to conversion tools from one format to the standard in readiness for template 
outside of the system - coordinator role to support where possible
need wider network of support eg from government agencies to help provide resources to help format data etc

**Comment by cmspinto** [comment]
An excel template? See current one for [VME](http://www.ices.dk/data/Documents/VME/VME_Reporting_Format_Template.xlsm)

**Comment by LynnHeeley** [comment]
scripts or tools are out of scope of the ICES system - however, an excel template will be provided and a human readable validation output on upload.

**Comment by LynnHeeley** [comment]
duplicate of #7  which is 'in the ICES bag' so labelled applied here - @neil-ices-dk  please check

**Comment by neil-ices-dk** [comment]
dependent on #223 therefore will be for start of October

**Comment by pcrjoana** [comment]
The template is almost finalised. @NikkiTaylorJNCC to take a look at the field descriptions and conditional fields.
Attached is the template version without the "Export to xml" function. The reason for this is because GitHub doesn't support .xlsm files. The full version was sent to Nikki, and is available from Joana and/or @cmspinto upon request.
[Cetaceans_data_template_v06.xlsx](https://github.com/ices-tools-dev/cetaceans/files/7553680/Cetaceans_data_template_v06.xlsx)

**Comment by neil-ices-dk** [comment]
v1.9 (25 NOV 2021) is uploaded to website
https://www.ices.dk/data/Documents/Cetaceans/Cetaceans_data_template.zip

**Comment by neil-ices-dk** [comment]
v1.10 (03 DEC 2021) will be finalised with changes to vocab and uploaded @pcrjoana and @neil-ices-dk to upload

**Comment by neil-ices-dk** [comment]
@NikkiTaylorJNCC 
v1.10 (07 DEC 2021) now released on web 
[https://www.ices.dk/data/Documents/Cetaceans/Cetaceans_data_template.zip](https://www.ices.dk/data/Documents/Cetaceans/Cetaceans_data_template.zip)
